2019-07-19 18:34:12,832 - ERROR - % - 116 - HTTPConnectionPool(host='mobile-api-test.sandanapp.com', port=80): Max retries exceeded with url: /component/upload2/uploadByForm/dynamic (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1085327f0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
2019-07-19 18:47:44,745 - ERROR - % - 165 - ConnectionError -- please wait 3 seconds
